Seed database

dwmorris11 opened the pull request.

dwmorris11 committed:
Set up linter.
Started writing dynamic seeding script.

dwmorris11 committed:
Generated dynamicseed and seeded the database with 500 lorem-ipsum experience entries.

chelseaSchmidt reviewed (changes requested):
Hey Dustin, here are the things I found:

- I think a css-loader is required in your webpack config, instead of the link in your HTML page. Only the bundle.js file will be available to the proxy server, so the bundle.js file needs to incorporate any css styling instead of using a link tag.

- The opening html tag is missing from the HTML page (I think !DOCTYPE html is a stand-alone tag and doesn't replace an opening html tag, at least as far as I know)

- I can't locate a bundle.js file in either the .gitignore file names or in your committed files. This might mean that your webpack build is failing. Are you seeing a bundle.js file get generated when you run webpack?

I'll hold off on feedback on the database aspects, as I'm not familiar with the libraries you're using - looks cool though!

dwmorris11 committed:
https://trello.com/c/USaM6BlN/14-update-html-to-remove-css-and-place-in-webpack

Added CSS to webpack config
Removed CSS from html
Updated .gitignore to include bundle.js
Ran webpack to create a bundle.js

dwmorris11 committed:
Modified index.html to have a beginning html tag.

dwmorris11 commented:
I wasn't working on the client side yet so I didn't run the webpack.  I updated webpack with the same css loaders that you used and ran the webpack to create a bundle.js.  I put the bundle.js in .gitignore.  I added the beginning <html> tag as I am not sure if it is a requirement either.
